src/svg/graphic.ts

```diff
@@ -185,7 +186,7 @@ export function brushSVGPath(el: Path, scope: BrushScope) {
         let svgPathBuilder = elExt.__svgPathBuilder;
         if (elExt.__svgPathVersion !== pathVersion
             || !svgPathBuilder
-            || strokePercent < 1
+            || el.style.strokePercent !== elExt.__svgPathStrokePercent
```

Comment:
Can use variable `strokePercent` instead of `el.style.strokePercent` here and following.

Reply:
Sure, I have changed the code to use 'strokePercent' directly.